pkg/manifest/installer.go

```diff
@@ -328,35 +355,43 @@ func applyManifest(componentName name.ComponentName, manifestStr string, version
 			}
 		}
 
-		stdoutCRD, stderrCRD, err = kubectl.Apply(opts.DryRun, opts.Verbose, opts.Kubeconfig, opts.Context, namespace, mcrd, extraArgs...)
+		stdoutCRD, stderrCRD, err := kubectl.Apply(opts.DryRun, opts.Verbose, opts.Kubeconfig, opts.Context, namespace, mcrd, extraArgs...)
+		stdout += "\n" + stdoutCRD
+		stderr += "\n" + stderrCRD
 		if err != nil {
-			// Not all Istio components are robust to not yet created CRDs.
-			if err := waitForCRDs(objects, opts.DryRun); err != nil {
-				return &ComponentApplyOutput{
-					Stdout: stdoutCRD,
-					Stderr: stderrCRD,
-					Err:    err,
-				}
+			return &ComponentApplyOutput{
+				Stdout: stdout,
+				Stderr: stderr,
+				Err:    err,
+			}
+		}
+		// Not all Istio components are robust to not yet created CRDs.
+		if err := waitForCRDs(objects, opts.DryRun); err != nil {
+			return &ComponentApplyOutput{
+				Stdout: stdout,
+				Stderr: stderr,
+				Err:    err,
```

Comment:
This block appears to be a duplicate of above. Would it be possible to create a helper here to avoid the duplication? It will also shorten this long function.

Reply:
Fixed.